extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -319,9 +319,13 @@ export class AmpImageViewer extends AMP.BaseElement {
         width: 0,
         height: 0,
       });
-      st.toggle(ampImg, false);
+      st.toggle(img, false);
       this.element.appendChild(this.image_);
-      return ampImg.getImpl().then((impl) => {
+      if (img.tagName === 'IMG') {
+        propagateAttributes(ARIA_ATTRIBUTES, img, this.image_);
+        return Promise.resolve();
+      }
+      return img.getImpl().then((impl) => {
         propagateAttributes(ARIA_ATTRIBUTES, impl.element, this.image_);
       });
```

Comment:
For your consideration, this might make it a little clearer that the same thing is happening in both cases

```suggestion
      const imgPromise = img.tagName === 'IMG')
        ? Promise.resolve(img)
        : img.getImpl().then((impl) => impl.element);
      return imgPromise.then((imgEl) => {
        propagateAttributes(ARIA_ATTRIBUTES, imgEl, this.image_);
      });
```